### PR TITLE
[CNCF Graduation]: Clarify Product Security group Membership

### PR DESCRIPTION
**Pull request by kfaseela**
Document organization affiliation of the individuals on the security alias.

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ Information about supported Istio versions can be found on the
 ## Reporting a Vulnerability
 
 Instructions for reporting a vulnerability can be found on the
-[Istio Security Vulnerabilities] page.
+[Istio Security Vulnerabilities] page. The Istio Product Security Working Group receives
+vulnerability and security issue reports. For information on who makes up that group,
+please refer to the [Early Disclosure Membership].
```
Comment (ericvn) [inline .github/SECURITY.md:13]: My question here is that the membership page mentions companies who are on the mailing list. And this mentions those on the Product Security Working Group (PSWG), which to me might be considered here: https://github.com/istio/community/blob/master/org/teams.yaml#L221-L236 (based on the yaml today). The only thing I do see is bullet #7 which says each company must have one person on the PSWG, so we have a subset.

Maybe we could change `who makes up` to which companies make up`.

Comment (kfaseela) [reply .github/SECURITY.md:13]: How about ```The Istio Product Security Working Group receives
vulnerability and security issue reports, and the company affiliation of the members of the group can be
found at [Early Disclosure Membership].```

Comment (kfaseela) [reply .github/SECURITY.md:13]: As per @GregHanson the companies are consistent between WG and early disclosure list, would like him to comment.

Comment (GregHanson) [reply .github/SECURITY.md:13]: @ericvn currently, company affiliation of members on the PSWG are correctly represented in the Early Disclosure Membership -  no more, no less.  Meaning all current members on the team today are the only companies listed there. 

cc @jacob-delgado who may have historical knowledge of how members there have been represented on the list

> But not sure that there are a large number of people who desire to be on the PSWG whose company would not want to also be on the list.

There is a time commitment expected of members on PSWG, which is the cost of getting your company name added to the Early Disclosure Membership list.

**Review by linsun — COMMENTED** [review]
This LGTM, happy to approve if there is no major concerns

File: .github/SECURITY.md (hunk 1 above)
Comment (linsun) [reply .github/SECURITY.md]: While what we have in not entirely accurate but it is close enough for CNCF graduation purpose imho.   The reason is in order to be on the early disclosure, you have to serve on the PSW working group.

**Review by ericvn — APPROVED** [review]
LGTM after change and discussion.

**Pull request merged**